app/views/admin/poll/questions/answers/documents.html.erb

```diff
@@ -0,0 +1,52 @@
+<%= back_link_to %>
+
+<h2><%= t("admin.questions.show.answers.documents_list") %></h2>
+
+<p><%= link_to @question.poll.name, admin_poll_path(@question.poll) %> > <%= link_to @question.title, admin_question_path(@question) %> > <%= @answer.title %></p>
+
+<div class="poll-question-answer-form">
+  <%= form_for(@answer, url: admin_question_answer_path(@question, @answer)) do |f| %>
+
+    <%= render 'shared/errors', resource: @answer %>
+
+    <%= f.hidden_field :question_id, value: @question.id %>
+
+    <div class="row">
+      <div class="small-12 column">
+        <div class="documents small-12">
+          <%= render 'documents/nested_documents', documentable: @answer, f: f %>
+        </div>
+
+        <div class="row">
+          <div class="actions small-12 medium-4 margin-top">
+            <%= f.submit(class: "button expanded", value: t("shared.save")) %>
+          </div>
+        </div>
+      </div>
+    </div>
+  <% end %>
+
+  <% if @answer.documents.present? %>
+    <table>
+      <tr>
+        <th scope="col"><%= t("admin.questions.show.answers.document_title") %></th>
+        <th scope="col"><%= t("admin.questions.show.answers.document_actions") %></th>
```

Comment:
here you can add `class="text-right"` (all the table actions on admin are there) 😄